webapp/components/hooks/useIsInRoute.js

```diff
@@ -0,0 +1,15 @@
+import { matchPath, useLocation } from 'react-router'
+import { designerModules, appModuleUri } from '@webapp/app/appModules'
+
+export const useIsInRoute = (path) => {
+  const { pathname } = useLocation()
+  return matchPath(pathname, path)
```

Comment:
does this work? I see https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/matchPath.md the second argument is an object. 
I thought this should be ```return matchPath(pathname, {path})```